internal/tools/tools.go

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build tools
```

Comment:
This was added due to the linter checking that this newer method is included (though it's not actually implemented, so we added a directive, instead of replacing the one below):

https://go.googlesource.com/proposal/+/master/design/draft-gobuild.md

```
We present a possible plan to transition from the current // +build lines for build tag selection to new //go:build lines that use standard boolean expression syntax.
```

Additional reference: https://golang.org/doc/go1.16

```
The new go/build/constraint package parses build constraint lines, both the original // +build syntax and the //go:build syntax that will be introduced in Go 1.17. This package exists so that tools built with Go 1.16 will be able to process Go 1.17 source code. See https://golang.org/design/draft-gobuild for details about the build constraint syntaxes and the planned transition to the //go:build syntax. Note that //go:build lines are not supported in Go 1.16 and should not be introduced into Go programs yet.
```